.changeset/unlucky-forks-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql": patch
+---
+
+Adds filtering on nested read operations
```

Comment:
Is this the correct changelog for this? 

It sounds like we are adding a feature, but this is a bugfix isn't it?